demisto_sdk/commands/common/schemas/layoutrule.yml

```diff
@@ -0,0 +1,50 @@
+type: map
```

Comment:
please update constants.SCHEMA_TO_REGEX in order to use the schema, and add a UT in structure_test.py